cachalot/apps.py

```diff
@@ -10,18 +9,6 @@
     SUPPORTED_ONLY)
 
 
-@register(Tags.compatibility)
-def check_django_version(app_configs, **kwargs):
-    if not (2, 0) <= django_version < (3, 2):
-        return [Error(
```

Comment:
Replace this with a warning. I agree an error is stupid, but I still highly suggest that at least a warning is mentioned. After every minor version update, Subquery always breaks in cachalot so a warning is still necessary. Ref my comments about async